### ISSUE TITLE
Implement additional events as requested (IRC command parsing)

### ISSUE THREAD

**Issue by numberoverzero** [numberoverzero/bottom#5]
Implement additional IRC commands and events as requested by users.

**Comment by AMorporkian** [comment]
Might want to modify that milestone again.

**Comment by numberoverzero** [comment]
Yeah.  Let's go with "Implement additional IRC commands and events as requested by users" and I'll work on an improvement to the README today.

**Comment by numberoverzero** [comment]
I can't believe I didn't celebrate this issue's 10 year anniversary, what a missed opportunity 😢

I'm going to close this.  Hopefully others will consider opening PRs to support new events ❤
